axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -1258,6 +1258,98 @@
           "address": "0x9186233C327329408a9034F22407244421cADbEF"
```

Comment:
PR titles should be more specific and meaningful, say
`feat(evm): add ability to update hyperliquid block sizes`